IRremoteESP8266.cpp

```diff
@@ -33,85 +33,99 @@
 // These versions of MATCH, MATCH_MARK, and MATCH_SPACE are only for debugging.
 // To use them, set DEBUG in IRremoteInt.h
 // Normally macros are used for efficiency
+
+bool  MATCH_MARK_NOLOG(int measured_ticks, int desired) {
+  return  measured_ticks >= TICKS_LOW(desired) && measured_ticks <= TICKS_HIGH(desired+MARK_EXCESS);
+}
+
+bool  MATCH_SPACE_NOLOG(int measured_ticks, int desired) {
+  return   measured_ticks >= TICKS_LOW(desired-MARK_EXCESS) && measured_ticks <= TICKS_HIGH(desired);
+}
+
+#define DEBUG true 
 #ifdef DEBUG
-int MATCH(int measured, int desired) {
-  Serial.print("Testing: ");
-  Serial.print(TICKS_LOW(desired), DEC);
-  Serial.print(" <= ");
-  Serial.print(measured, DEC);
-  Serial.print(" <= ");
-  Serial.println(TICKS_HIGH(desired), DEC);
-  return measured >= TICKS_LOW(desired) && measured <= TICKS_HIGH(desired);
+// Changes for just log when not matches to have a clean debug 
```

Comment:
this comment is a changelog for commit msg and not relevant to be in the code.
anyway why did you change the debug function. the original debug was to output debug even if matches or not matched and not just not matched case.